src/app.rs

```diff
@@ -207,6 +207,12 @@ pub fn build_app() -> App<'static, 'static> {
                 .hidden_short_help(true),
         )
         .arg(arg("pattern"))
+        .arg(
+            arg("path-separator")
+                .takes_value(true)
+                .long("path-separator")
+                .number_of_values(1),
```

Comment:
is this needed?

Reply:
Honestly, I don't really know what I am doing here; I am inexperienced with clap and tried to find something that works ok.